### PR TITLE
Possible fix for #83

### PR DESCRIPTION
**Pull request by thirtytwobits**

### CODE REVIEW

File: pydsdl/_dsdl_definition.py [hunk 1]
```diff
@@ -48,7 +48,7 @@ def __init__(self, file_path: Path, root_namespace_path: Path):
         relative_path = str(
             os.path.join(
                 os.path.split(self._root_namespace_path)[-1],
```
Comment (SchoberMJ) [inline pydsdl/_dsdl_definition.py:50]: Is the intent for this to be the minimal change necessary to fix the bug? 
It looks like this is still mixing os.path and Path a bit. Would we not want to use 
`relative_path = str(self._root_namespace_path.stem / self._file_path.relative_to(self._root_namespace_path)`?

Comment (SchoberMJ) [reply pydsdl/_dsdl_definition.py:50]: I suppose this could also be covered in Issue #84

**Review by SchoberMJ — COMMENTED** [review]
Looks good to me assuming it is a minimal change to get a use-case working again.

File: pydsdl/_dsdl_definition.py (hunk 1 above)
Comment (thirtytwobits) [reply pydsdl/_dsdl_definition.py:50]: Correct. #83 fixes the immediate issue. #84 removes the underlying reason for the defect.

**Review by pavel-kirienko — COMMENTED** [review]
This is fine, but the workflow is dysfunctional. Can we please either remove it or fix it before merging? I don't want to introduce a new CI configuration that is known to be incorrect. In general, I think we would benefit if we migrated from AppVeyor to GitHub Actions across all of our repos, if only for the sake of consistency.

**Review by pavel-kirienko — APPROVED** [review]

**Pull request merged**